### ISSUE TITLE
Failure when running validator using RuntimeBatchRequest

### ISSUE THREAD

**Issue by shpolina** [fivetran/great_expectations#2821]
**Describe the bug**
I am following this [v3 example](https://docs.greatexpectations.io/en/latest/guides/how_to_guides/creating_batches/how_to_load_a_database_table_view_or_query_result_as_a_batch.html)  to construct a batch request but getting an error when running `validator.active_batch.head()` due to invalid SQL that is being generated.

The error:
```(psycopg2.errors.SyntaxError) syntax error at or near "SELECT"
LINE 2: FROM SELECT * FROM my_table_name
             ^

[SQL: SELECT * 
FROM SELECT * FROM my_table_name
 LIMIT 5]
(Background on this error at: http://sqlalche.me/e/f405)
```

**To Reproduce**
1. my datasoure config that connects to Redshift db:
```
rs_datasource:
    class_name: Datasource
    module_name: great_expectations.datasource
    execution_engine:
      class_name: SqlAlchemyExecutionEngine
      module_name: great_expectations.execution_engine
      connection_string: ${my_connection_string}
      create_temp_table: False
    data_connectors:
      my_dc:
        module_name: great_expectations.datasource.data_connector
        class_name: RuntimeDataConnector
        batch_identifiers:
        - some_key
```
2. Create batch request:
```python
batch_request = RuntimeBatchRequest(
	datasource_name="rs_datasource",
	data_connector_name="my_dc",
	data_asset_name=my_table_name,
	runtime_parameters={"query": "SELECT * FROM my_table_name"},
	batch_identifiers={"some_key": "my_key"},
	batch_spec_passthrough={"create_temp_table": False},  
)
```
3. run
```python
my_validator = context.get_validator(
    batch_request=batch_request,
    expectation_suite=suite
)
my_validator.active_batch.head()
````

**Expected behavior**
Display rows from Redshift table.

**Environment (please complete the following information):**
 - Operating System: Windows
 - Great Expectations Version: 0.13.19

**Comment by eugmandel** [comment]
@shpolina Thank you for reporting this!

**Comment by talagluck** [comment]
Hi @shpolina - thanks for reporting this, and apologies for delays! I believe this is resolved by #3508, so closing this for now.
